var/spack/repos/builtin/packages/spectre/package.py

```diff
@@ -0,0 +1,200 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Spectre(CMakePackage):
+    """SpECTRE is an open-source code for multi-scale, multi-physics problems in
```

Comment:
It's customary to have a package description that has a title, which is a single (not-too-long line) that ends in a period, followed by an empty line. Some of Spack's tools output only the first line of the description.